src/tetris/field/field.ts

```diff
@@ -126,6 +146,20 @@ export default class Field {
 			|| position.x >= this.width
 			|| position.y >= this.height);
 	}
+
+	public reset(): void {
+		for (const blockRow of this._blockRows)  {
```

Comment:
nesting level isn't that nice.
An alternative would be to use `forEach` and/or split it to multiple functions.
Splitting it up would as a sideline increase readability,

Reply:
I don't think that this is necessary here, because the code reads straightforward. It just iterates over all blocks, how would you split this?

Reply:
`this._blockRows.forEach(blockRow => blockRow.forEach(block => block.destroy()));`

Reply:
That's exactly what I wanted to avoid. I think this would rather decrease the readability.

Reply:
okay